src/content/docs/it/tutorial/2-pages/1.mdx

```diff
@@ -0,0 +1,162 @@
+    Il tuo editor potrebbe mostrare un cerchio bianco pieno sull'etichetta della scheda per questo file. Questo significa che il file non è ancora stato salvato. Nel menu File in VS Code, abilita "Salvataggio automatico" e non dovresti più aver bisogno di salvare manualmente alcun file.
+    :::
+
+4. Aggiungi `/about` alla fine dell'URL di anteprima del tuo sito web nella barra degli indirizzi e verifica di poter vedere una pagina caricarsi lì. (es. `http://localhost:4321/about`)
+</Steps>
+
+In questo momento, la tua pagina "About" dovrebbe apparire esattamente come la prima pagina, ma stiamo per cambiarlo!
+
+## Modifica la tua pagina
+
+Modifica il contenuto HTML per rendere questa pagina su di te.
+
+Per cambiare o aggiungere più contenuto alla tua pagina About, aggiungi più tag di elementi HTML contenenti contenuto. Puoi copiare e incollare il codice HTML qui sotto tra i tag `<body></body>` esistenti, o creare il tuo.
+
+```astro title="src/pages/about.astro" ins={3-8} del={2}
+<body>
+  <h1>Il mio sito Astro</h1>
+  <h1>Su di me</h1>
+  <h2>... e il mio nuovo sito Astro!</h2>
+
+  <p>Sto seguendo il tutorial introduttivo di Astro. Questa è la seconda pagina sul mio sito web, ed è la prima che ho costruito da solo!</p>
+
+  <p>Questo sito si aggiornerà man mano che completerò altre parti del tutorial, quindi continua a controllare e vedi come sta andando il mio percorso!</p>
+</body>
+```
+
+Ora, visita di nuovo la tua pagina `/about` nella scheda del tuo browser, e dovresti vedere il tuo contenuto aggiornato.
+
+## Aggiungi link di navigazione
+
+Per rendere più facile visualizzare in anteprima tutte le tue pagine, aggiungi link di navigazione di pagina HTML prima del tuo `<h1>` nella parte superiore di entrambe le tue pagine (`index.astro` e `about.astro`):
+
+```astro title="src/pages/about.astro" ins={1-2}
+<a href="/">Home</a>
+<a href="/about/">About</a>
+
+<h1>Su di me</h1>
+<h2>... e il mio nuovo sito Astro!</h2>
+```
+
+Verifica di poter cliccare su questi link per spostarti avanti e indietro tra le pagine sul tuo sito.
+
+:::note
+A differenza di molti framework, Astro utilizza elementi `<a>` HTML standard per navigare tra le pagine (chiamate anche _rotte_), con i tradizionali aggiornamenti di pagina.
+:::
+
+<Box icon="puzzle-piece">
+## Prova tu stesso - Aggiungi una pagina Blog
+
+Aggiungi una terza pagina `blog.astro` al tuo sito, seguendo [gli stessi passaggi di sopra](#crea-un-nuovo-file-astro).
+
+(Non dimenticare di aggiungere un terzo link di navigazione a ogni pagina.)
+
+<details>
+<summary>Mostrami i passaggi.</summary>
+<Steps>
+1. Crea un nuovo file in `src/pages/blog.astro`.
+2. Copia l'intero contenuto di `index.astro` e incollalo in `blog.astro`.
+3. [Aggiungi un terzo link di navigazione](#aggiungi-link-di-navigazione) alla parte superiore di ogni pagina:
+</Steps>
+
+```astro title="src/pages/index.astro" ins={4}
+<body>
+  <a href="/">Home</a>
+  <a href="/about/">About</a>
+  <a href="/blog/">Blog</a>
```

Comment:
Same question as my previous comment.

Note: the same question can occur at various places after that so I'm not going to continue mentioning it but if it's updated here, a few more places should be updated too.